src/database.cpp

```diff
@@ -156,7 +183,17 @@ std::string Database::open(bool wait, bool memory) {
   while (true) {
     char *fail;
     ret = sqlite3_exec(imp->db, schema_sql, 0, 0, &fail);
-    if (ret == SQLITE_OK) break;
+    if (ret == SQLITE_OK) {
+      const char *get_version = "select (select count(row_id) from entropy), (select max(version) from schema);";
+      const char *set_version = "insert or ignore into schema(version) values(" SCHEMA_VERSION ");";
+      ret = sqlite3_exec(imp->db, get_version, &schema_cb, 0, 0);
+      if (ret == SQLITE_OK) {
+        sqlite3_exec(imp->db, set_version, 0, 0, 0);
+        break;
+      } else {
+        return "produced by an incompatible verison of wake; remove it.";
```

Comment:
You do not need to run `wake --init` in v0.18. If it finds a wit-lock.json, it will autoinit a wake.db.

Reply:
The error FYI is: 
```
Failed to open wake.db: produced by an incompatible verison of wake; remove it.
```

Reply:
> You do not need to run wake --init in v0.18. If it finds a wit-lock.json, it will autoinit a wake.db.

Neat, I remember this now that you mention it, but totally forgot.

And thanks, yes, I agree that error message is pretty clear.